### ISSUE TITLE
Resolve latex format warnings in foil.tex

### ISSUE THREAD

**Issue by YenPeiChen07** [yungyuc/bezierfoil#7]
When running linter in CI, it shows the following warnings for `foil.tex`:

```
Warning 24 in foil.tex line 82: Delete this space to maintain correct pagereferences.
make: *** [Makefile:134: tex] Error 2
    \label{f:airfoil_shape_image}  
   ^
Warning 24 in foil.tex line 160: Delete this space to maintain correct pagereferences.
    \label{f:cubic_bezier_curve}  
   ^
Warning 12 in foil.tex line 197: Interword spacing (`\ ') should perhaps be used.
where $A \in \mathbb{R}^{m \times n}$ and $b \in \mathbb{R}^m$. when $m>n$, 
```

Currently, these two warnings was [suppressed in `foil.tex`](https://github.com/yungyuc/bezierfoil/blob/d603f964db1a7dbb40ef929064d7773d674ff745/foil.tex#L8).

**Comment by yungyuc** [comment]
@Gene0315 please take a look and reformat to fix it.

**Comment by yungyuc** [comment]
Resolved by PR #11 .  The remaining suppressions are necessary.
